tests/unit/components/plates/WellItem.spec.js

```diff
@@ -83,26 +83,26 @@ describe('Well.vue', () => {
   })
 
   describe('#tooltip', () => {
-    it('will display the materials requests name', () => {
+    it('will display the requests name', () => {
       const title = wrapper.find('title')
-      const expected = well.well_info.materials.map((m) => m.sample_name).join(', ')
+      const expected = well.well_info.requests.map((r) => r.sample_name).join(', ')
```

Comment:
Do you think this is a code smell? As you are mimicking the actual method in the component.

Reply:
Yeah this test probably isnt the best. Will replace with the the component method